src/_guides/language/effective-dart/design.md

```diff
@@ -98,24 +99,26 @@ When in doubt about naming, write some code that uses your API, and try to read
 it like a sentence.
 
 {:.good-style}
+<?code-excerpt "misc/lib/effective_dart/design_good.dart (code-like-prose)"?>
 {% prettify dart %}
 // "If errors is empty..."
-if (errors.isEmpty) ...
+if (errors.isEmpty) { ... }
 
-// "Hey, _subscription, cancel!"
-_subscription.cancel();
+// "Hey, subscription, cancel!"
+subscription.cancel();
```

Comment:
For consistency, I dropped the leading `_` because we use `subscription` elsewhere in the page.